front-end/src/app/shared/services/fec-api.service.ts

```diff
@@ -60,48 +59,9 @@ export class FecApiService {
     if (!committeeId) {
       throw new Error('No Committee Id provided.');
     }
-    const headers = this.getHeaders();
```

Comment:
We still need this code right?  we just need to configure the endpoints it hits to be mocks when the system is configured to test